content/en/dashboards/guide/is-read-only-deprecation.md

```diff
@@ -0,0 +1,31 @@
+---
+title: API Deprecation - Removing is_read_only attribute
+---
+
+## Overview
+
+On November 30, 2024, Datadog will remove support for the `is_read_only` attribute in the Dashboards API's. This is part of our ongoing efforts to enhance our services and streamline our offerings. Datadog is encouraging customers who manage Dashboards with the API directly to transition to `restricted_roles` or Restriction Policies. 
+
+1. `restricted_roles` allow Dashboard owners to assign users with roles with certain permissions.
```

Comment:
Hmm, that shouldn't be the case. Let me file a PR removing that.

Reply:
It looks like this API doc change was autogenerated when we deprecated the attributes in the API spec. 

The eventual goal is to have people on Restriction Policies for Dashboards. Unfortunately, we can't do that in one go here because of Restriction Policies for Dashboards technically being in beta 😅 .